desktop/apps/home/routes.coffee

```diff
@@ -83,6 +83,7 @@ fetchMetaphysicsData = (req)->
 
       res.locals.sd.HERO_UNITS = heroUnits
       res.locals.sd.USER_HOME_PAGE = home_page.artwork_modules
+      res.locals.sd.HIGHLIGHT_HEADER_SEARCH = req.query?.highlight_header_search
```

Comment:
@mzikherman I check for the query parameter here, but it feels like it should be a little more global. Is there any way to do that?